src/server/routes/apiv3/personal-setting.js

```diff
@@ -165,9 +166,8 @@ module.exports = (crowi) => {
       const updatedUser = await user.save();
       req.i18n.changeLanguage(req.body.lang);
       return res.apiv3({ updatedUser });
-    }
-    catch (err) {
-      logger.error(err);
+    } catch (err) {
```

Comment:
This line causes `brace-style` eslint error. Please invoke `npm run lint:js` and check other eslint errors.